plugins/inputs/docker/docker.go

```diff
@@ -141,6 +160,21 @@ func (d *Docker) Description() string {
 	return "Read metrics about docker containers"
 }
 
+// Config Validation
+func (d *Docker) Init() error {
+	err := choice.CheckSlice(d.PerDeviceInclude, containerMetricClasses)
```

Comment:
What happens if `p.PerDeviceInclude` is empty? The check will not trigger. However, other plugins assume that all options are set if the include list is empty. You might also want to do this. In any case, please document the behavior in `sampleConfig` and the README.